types/fingerpose/index.d.ts

```diff
@@ -109,7 +109,7 @@ export class GestureEstimator {
         keypoints3D: Keypoint3D[],
         minScore: number,
     ): {
-        poseData: GestureDescription[];
+        poseData: Array<[string, string, string]>;
```

Comment:
please consider:
```suggestion
        poseData: Array<[name: string, curl: string, direction: string]>;
```
named tuples are widely supported now.
<img width="698" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/14539/fa9d9295-5b2b-4d2a-ae99-e12ae54ca637">